rascaline/src/calculators/lode/radial_integral/spline.rs

```diff
@@ -123,18 +123,17 @@ mod tests {
 
     #[test]
     fn high_accuracy() {
-        // Check that even with high accuracy and large domain MAX_SPLINE_SIZE
-        // is enough
+        // Check that even with high accuracy and large domain MAX_SPLINE_SIZE is enough
         let parameters = LodeRadialIntegralSplineParameters {
             max_radial: 15,
             max_angular: 10,
-            cutoff: 12.0,
+            k_cutoff: 10.0,
         };
 
         let gto = LodeRadialIntegralGto::new(LodeRadialIntegralGtoParameters {
             max_radial: parameters.max_radial,
             max_angular: parameters.max_angular,
-            cutoff: parameters.cutoff,
+            cutoff: 5.0,
```

Comment:
I changed this value here to a different one as `k_cutoff`. Otherwise one might be confused why the real and the k-space cutoff has to be same.

Reply:
👍